src/state/endpoints/tests/reducer.test.js

```diff
@@ -0,0 +1,79 @@
+import deepfreeze from 'deepfreeze';
```

Comment:
Is naming of these files a convention from create-react-app ? Seems redundant with `test` twice in the path